src/lib/cache/subCache.cpp

```diff
@@ -734,6 +737,8 @@ void subCacheItemInsert
   cSubP->expression.geometry   = geometry;
   cSubP->expression.coords     = coords;
   cSubP->expression.georel     = georel;
+  cSubP->httpInfo.url          = url.c_str();
+  cSubP->httpInfo.verb         = POST;  // FIXME PR: Unhardwire
```

Comment:
Yes, I believe `httpInfo*` might be good as argument to this function.
I haven't really thought this over though, been concentrating on other stuff